include/pocl.h

```diff
@@ -516,12 +522,7 @@ struct _cl_command_node
   cl_device_id device;
   /* The index of the targeted device in the **program** device list. */
   unsigned program_device_i;
-  /*
-   * -1: command_failed
-   * 0: not ready
-   * 1: ready
-   */
-  cl_int ready;
+  command_node_state node_state;
```

Comment:
```suggestion
  pocl_command_node_state node_state;
```
Due to lack of namespaces, best add name prefixes.